drivers/soundwire/cadence_master.c

```diff
@@ -306,8 +306,8 @@ _cdns_xfer_msg(struct sdw_cdns *cdns, struct sdw_msg *msg, int cmd,
 	return cdns_fill_msg_resp(cdns, msg, count, offset);
 }
 
-static enum sdw_command_response cdns_program_scp_addr(
-			struct sdw_cdns *cdns, struct sdw_msg *msg)
+static enum sdw_command_response
+cdns_program_scp_addr(struct sdw_cdns *cdns, struct sdw_msg *msg)
```

Comment:
@plbossart this looks odd should the function name be on the previous line?

Reply:
it's common practice, even in this file, to put the type first when the entire function doesn't fit on one line.

Reply:
ok